Also add ATen/core tests to oss CI

smessmer opened the pull request:
Summary: -

Differential Revision: D9070030

ezyang commented on aten/CMakeLists.txt:
```diff
@@ -135,6 +135,9 @@ list(APPEND ATen_CPU_INCLUDE
 add_subdirectory(src/ATen)
 
+file(GLOB tmp ${CMAKE_CURRENT_SOURCE_DIR}/src/ATen/test/core/*.cpp)
+set(Caffe2_CPU_TEST_SRCS ${Caffe2_CPU_TEST_SRCS} ${tmp})
```
And then you wouldn't need two cmake files, you can just put it all in `ATen/core/CMakeLists.txt`

ezyang replied:
Also, why are you monkeying around with `Caffe2_CPU_TEST_SRCS` directly?

smessmer replied:
don't have a strong opinion, moved it to the same directory. Also not monkeying around anymore ;)

ezyang commented on aten/CMakeLists.txt:
```diff
@@ -135,6 +135,9 @@ list(APPEND ATen_CPU_INCLUDE
 add_subdirectory(src/ATen)
 
+file(GLOB ATen_CORE_TEST_SRCS "*_test.cpp")
+EXCLUDE(ATen_CPU_SRCS "${ATen_CPU_SRCS}" ${ATen_CORE_TEST_SRCS})
```
This looks like copy-paste. Can't you propagate it from aten/src/ATen/CMakeLists.txt by having that cmake set it to PARENT_SCOPE?

smessmer replied:
fixed

smessmer committed:
Also add ATen/core tests to oss CI (#10029)

Summary:
Pull Request resolved: https://github.com/pytorch/pytorch/pull/10029

-

Differential Revision: D9070030

fbshipit-source-id: 305e4d890fc73e622c1016d1f9f1985ea792a924
